lib/rspec/bash/util/call_log_argument_list_matcher.rb

```diff
@@ -0,0 +1,33 @@
+require 'rspec/mocks/argument_list_matcher'
+include RSpec::Mocks
+include RSpec::Mocks::ArgumentMatchers
+
+module Rspec
+  module Bash
+    module Util
+      class CallLogArgumentListMatcher < ArgumentListMatcher
+        alias parent_args_match? args_match?
```

Comment:
I noticed you switched this from alias_matcher to alias. Why? (I'm still trying to understand the difference but everything I've seen recommends to use alias_matcher)

http://blog.bigbinary.com/2012/01/08/alias-vs-alias-method.html